### ISSUE TITLE
Problems instaling GPy for Python3 on ubuntu

### ISSUE THREAD

**Issue by MartijnZ** [SheffieldML/GPy#284]
Hi,

I've tried to install GPy on Ubuntu 14.04 for Python3, but with no success. Find below the different attempts. Do you have any hints how to resolve this?

Thanks,

Martijn
### Using 'Ubuntu Hackers' solution described on the website

```
git clone https://github.com/SheffieldML/GPy.git
echo 'PYTHONPATH=$PYTHONPATH:~/SheffieldML' >> ~/.bashrc
```

Results in numpy not found
### Using setup.py

First download the library:

```
git clone https://github.com/SheffieldML/GPy.git
```

Installing:

```
sudo python3 setup.py install
```

That seems to work, but when I then do 

```
import GPy
```

I get:

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-c6d42d62ab4a> in <module>()
----> 1 import GPy

/usr/local/lib/python3.4/dist-packages/GPy-0.8.8-py3.4-linux-x86_64.egg/GPy/__init__.py in <module>()
      4 warnings.filterwarnings("ignore", category=DeprecationWarning)
      5 
----> 6 from . import core
      7 from .core.parameterization import transformations, priors
      8 constraints = transformations

/usr/local/lib/python3.4/dist-packages/GPy-0.8.8-py3.4-linux-x86_64.egg/GPy/core/__init__.py in <module>()
      8 
      9 from .gp import GP
---> 10 from .svgp import SVGP
     11 from .sparse_gp import SparseGP
     12 from .mapping import *

/usr/local/lib/python3.4/dist-packages/GPy-0.8.8-py3.4-linux-x86_64.egg/GPy/core/svgp.py in <module>()
      6 from .sparse_gp import SparseGP
      7 from .parameterization.param import Param
----> 8 from ..inference.latent_function_inference.svgp import SVGP as svgp_inf
      9 
     10 

/usr/local/lib/python3.4/dist-packages/GPy-0.8.8-py3.4-linux-x86_64.egg/GPy/inference/latent_function_inference/svgp.py in <module>()
      4 import numpy as np
      5 from .posterior import Posterior
----> 6 from scipy.linalg.blas import dgemm, dsymm, dtrmm
      7 
      8 class SVGP(LatentFunctionInference):

ImportError: cannot import name 'dtrmm'
```
### Using PIP:

`sudo pip3 install GPy`

results in:

```
      Command "/usr/bin/python3 -c "import setuptools, tokenize;__file__='/tmp/pip-build- 
      po6mo3o6/GPy/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', 
     '\n'), __file__, 'exec'))" install --record /tmp/pip-nn0t6jm1-record/install-record.txt --single-version-
     externally-managed --compile" failed with error code 1 in /tmp/pip-build-po6mo3o6/GPy
```

**Comment by mzwiessele** [comment]
What scipy version are you using? You need scipy>=0.16. Also using setup.py, you should switch to the devel branch:

$ git checkout devel
$ python setup.py develp

**Comment by MartijnZ** [comment]
Hi Max,

Thanks for your quick reply.

I indeed checked out the development branch. And I also tried to upgrade scipy to the newest version `sudo pip3 install scipy --upgrade`. However, inspecting the version it turns out that pip didn't upgrade to 0.13, the version I run is still 0.13. 

After some struggle I managed to upgrade to 0.16.1. Now `import GPy` does not result in errors.

Thanks again for the support.

Martijn
